### DISCUSSION TITLE
admin_api_key for sandbox

### DISCUSSION THREAD

**Discussion by ram33sul** [juspay/hyperswitch#5206] [Q&A] [answered]
Hi, I'm trying to implement Hyperswitch, but I'm having trouble finding the correct admin_api_key for the sandbox account. Can someone please help me with this? I've gone through the entire documentation and code base, but none of the keys are working for the sandbox account. Thanks!

**Comment by SanchithHegde** [comment, marked as answer]
Hey @ram33sul, the admin API key for the sandbox environment is only known to limited number of people within the team, it's not documented anywhere in our documentation. 

If you want to perform updates on your merchant account, merchant connector account, etc. on our hosted sandbox environment, you'd have to use the control center, you cannot use the admin API key to perform any operations on our hosted sandbox environment. If you absolutely need to test out operations using the admin API key, then your only option is to run Hyperswitch locally (or self-host it), where you'd have complete control over the admin API key.

Marking this discussion as resolved.
